prow/cmd/phony/README.md

```diff
@@ -0,0 +1,33 @@
+# Phony
+
+`phony` sends fake GitHub webhooks.
+
+## Running a GitHub event manager
+`phony` is most commonly used for testing [hook](../hook), but can be used for testing any
+externally exposed service configured to receive GitHub events.
+
+To get an idea of `phony`'s behavior, start a local instance of `hook` with
+this:
+```
+go run prow/cmd/hook/main.go --deck-url=localhost
```

Comment:
`deck-url` should be set to the URL of deck in production (e.g. https://prow.k8s.io), not the localhost.
Please also move this flag to its own line.